_includes/reactions.njk

```diff
@@ -0,0 +1,32 @@
+<div id="reactions-list">
```

Comment:
Some kind of comment about making sure this never gets included twice in a page would be great.

Reply:
done!